cuboid_muscle/settings_force.py

```diff
@@ -256,12 +256,12 @@ def callback_function(raw_data):
     print("length of muscle: ", length_of_muscle)
 
     if t == variables.dt_3D:
-      f = open("muscle_length.csv", "w")
+      f = open("muscle_length1.csv", "w")
```

Comment:
muscle_length1 is not a descriptive name:
Either use muscle_length for both callback functions or add the word prestretch and contraction to the file names